### PR TITLE
rollout apidoc

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

File: src/api-doc/api-doc.yaml [hunk 1]
```diff
@@ -4959,6 +4959,696 @@ paths:
...
+            application/json:
+              schema:
+                type: object
+                properties:
+                  responseCode:
+                    type: string
+                    example: OK
+                  message:
+                    type: string
+                    example: Your changes have been published
```
Comment (adithyadinesh0412) [inline src/api-doc/api-doc.yaml]: Can we have this message "Your changes have been saved" ? As we have a different API for publish? @priyanka-TL

File: src/api-doc/api-doc.yaml [hunk 2]
```diff
@@ -4959,6 +4959,696 @@ paths:
...
+                  result:
+                    type: object
+                    properties:
+                      id:
+                        type: number
+                        example: 1
```
Comment (adithyadinesh0412) [inline src/api-doc/api-doc.yaml:5572]: Please recheck this response structure. Looks like this is broken.  @priyanka-TL 
Screenshot 2024-11-26 at 2 15 13 PM

File: src/api-doc/api-doc.yaml
Comment (adithyadinesh0412) [inline src/api-doc/api-doc.yaml:1]: @priyanka-TL  Are we not changing the browse existing API?

**Review by adithyadinesh0412 — CHANGES_REQUESTED** [review]
Reviewed on 26-Nov 2.23PM

File: src/api-doc/api-doc.yaml (hunk 1 above)
Comment (priyanka-TL) [reply src/api-doc/api-doc.yaml]: It's the message provided by product

File: src/api-doc/api-doc.yaml (hunk 2 above)
Comment (priyanka-TL) [reply src/api-doc/api-doc.yaml:5572]: checked its working fine

**Review by adithyadinesh0412 — APPROVED** [review]
Approved 26 Nov - 6.16PM

**Pull request merged**